### DISCUSSION TITLE
Project Find/replace like VSCode

### DISCUSSION THREAD

**Discussion by thiagomajesk** [helix-editor/helix#11788] [Ideas]
I know that more recently the global search was merged and I was wondering if there are plans to keep improving on that, like adding a replace feature. I came across this little CLI tool the other day and Helix instantly came to mind: https://github.com/yassinebridi/serpl.

I think a better version of this would be a Zed-like multi buffer; but don’t know how feasible it would be. Anyway, could be a good place to start.

**Comment by kirawi** [comment]
https://github.com/helix-editor/helix/pull/4381

**Reply by thiagomajesk** [reply]
I’m aware of this PR, it’s been there for quite a while (one of the reasons why I’m sharing serpl here)

**Comment by nik-rev** [comment]
you may be interested in https://github.com/thomasschafer/scooter
